tidy3d/components/data/sim_data.py

```diff
@@ -369,6 +376,65 @@ def get_intensity(self, field_monitor_name: str) -> xr.DataArray:
             field_monitor_name=field_monitor_name, field_name="E", val="abs^2"
         )
 
+    @classmethod
+    def mnt_data_from_file(cls, fname: str, mnt_name: str, **parse_obj_kwargs) -> MonitorDataType:
+        """Loads a monitor data from .hdf5, or .hdf5.gz file.
```

Comment:
The check below (line 404) does not seem to allow `.hdf5.gz`. Does `pathlib.Path(fname)` internally extract `.gz` files?

Reply:
oh right, no the pathlib.path just is to check the extension but I'll make the docstring clearer that it only works for hdf5.

Reply:
I guess if we start gzipping simulation data files, we should still unzip them right after download.